public/schemas/applicant_v5.json.erb

```diff
@@ -15,8 +15,15 @@
         "has_partner_opponent": {
           "type": "boolean"
         },
+        "involvement_type": {
+          "type": ["string", "null"],
+          "enum": ["applicant", "nil"]
```

Comment:
This feels weird to me.

I'd assume non-required fields would be allowed to be null.

Should we add optional fields to the schema at all?